inc/api.php

```
@@ -409,3 +415,141 @@ function djz_prime_thumbnails_cache($ids) {
+        'posts_per_page' => 10,
+        'orderby' => 'date',
+        'order' => 'DESC',
+        'tax_query' => [[
+            'taxonomy' => 'product_cat',
+            'field'    => 'slug',
+            'terms'    => 'featured',
+            'operator' => 'NOT IN'
+        ]]
+    ]);
+
+    $best_sellers = $format_products([
```

PHPMD (2.15.0)

[error] The variable $best_sellers is not named in camelCase. (undefined)

(CamelCaseVariableName)